backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -935,12 +892,33 @@ internal class FunctionGenerationContext(val function: LLVMValueRef,
                 )
             }
         }
+
+        // If object is imported - access it via getter function.
+        if (isExternal(irClass)) {
+            val valueGetterName = irClass.objectInstanceGetterSymbolName
+            val valueGetterFunction = LLVMGetNamedFunction(context.llvmModule, valueGetterName) ?:
```

Comment:
Is `context.llvm.externalFunction` applicable here?

Reply:
Function above mentions in comments that it works with C++ functions and does related signext magic here, so decided to avoid using it here.

Reply:
It is actually used for Kotlin functions.